### ISSUE TITLE
Login- Error is not highlighted in red when typing a password not meet the requirements

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#8968]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Go to https://staging.new.expensify.com/
2. Start typing a new email account
3. Click Continue
4. Go back to your email and tap on link
5. Fill the password field with a value that does not comply with the rules
6. Click on "Set Password" button

## Expected Result:
The message below the field where the password is entered should be highlighted in red

## Actual Result:
The message below the field is not highlighted in red when a password that does not meet the requirements is entered

## Workaround:
Unknown

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.1.57.15

**Reproducible in staging?:** Yes

**Reproducible in production?:** No

**Email or phone of affected tester (no customers):** any

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/93399543/168123960-24cf01f2-c5f8-4a1d-998e-352739af0773.mp4

**Expensify/Expensify Issue URL:** 

**Issue reported by:** Applause

**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by bondydaa** [comment]
yep def broken, it might be related to this as well https://github.com/Expensify/App/issues/8440 cc @NikkiWines @parasharrajat should we combine these 2, I have idea if it's the same code or not but probably should be or at least have the same behaviors.

**Comment by thesahindia** [comment]
@bondydaa, I don't think it's related to #8440, also this isn't an issue we keeps the "Set password" button disabled until user enters a valid password, so basically there's no way for the user to submit a password like this.

Right now the button stays enabled because of the regression from #8680 and it already got logged [here](https://github.com/Expensify/App/issues/8962) so we can close this one.
